### DISCUSSION TITLE
Omit binary version in release.

### DISCUSSION THREAD

**Discussion by ghost** [goreleaser/goreleaser#2360] [Q&A] [answered]
👋 

## Goal
When using scripts to download the latest release of a project, it is cumbersome if the filename is versioned too. The release already carries the version so the filename can omit it. If its inside the filename, you must make two API calls, 1 to get the version, the second to download.

## Current behaviour
```
polaris_4.0.5_linux_amd64.tar.gz
```
Like [so](https://github.com/FairwindsOps/polaris/releases/tag/4.0.5).

## Wanted behaviour
```
polaris_linux_amd64.tar.gz
```
[.goreleaser.yml](https://github.com/FairwindsOps/polaris/blob/master/.goreleaser.yml)

## Question
How does one customise the filename to omit the version?

🙏

**Comment by caarlos0** [comment, marked as answer]
By setting archives name template..

https://goreleaser.com/customization/archive/

almost all (if not all) of the artifacts generated have some sort of name template field.
